doc/mp3_support.md

```diff
@@ -0,0 +1,22 @@
+# MP3 playback support in Julius
+
+Julius currently supports the playback of MP3 files instead of the original, low-quality wav files.
+
+If you have the high-quality MP3 files that were once available from the [Sierra Website](http://web.archive.org/web/20000303064239/www.caesar3.com/downloads.shtml), you can play them instead of the original files. Alternatively, you can play any MP3 file you like instead of the original music.
+
+In order to play the files, you must have the library `libmpg123-0` installed. For Windows, if you download [`SDL2_mixer`](https://www.libsdl.org/projects/SDL_mixer/), the library should be available as a DLL (`libmpg123-0.dll`) with the download. Simply copy it to the same directory from where you run Julius.
+
+For Linux and Apple, it's probably already been installed with `SDL2_mixer`, so nothing should be required.
```

Comment:
For macOS, libmpg123 is already packaged in the .app.

Reply:
And in the Framework as well? I left that text because perhaps there's a chance that `libmpg123` isn't packaged.

Reply:
Every dependency of SDL_Mixer (including libmpg123) is packaged in the .app when you use the Framework variant. I haven't used homebrew for packaging the .app bundle, so for that I don't know. Builds with homebrew are probably only going to work on that user's machine.

Reply:
On linux, depending on configuration (Gentoo comes to mind) libmpeg might not be installed... I guess.